frontend/src/components/housing/submission/SubmissionsNavigator.vue

```diff
@@ -255,85 +267,101 @@ watch(showCompleted, () => {
     v-if="showToggle"
     class="flex justify-content-end mr-3 align-items-center"
   >
-    <span class="app-primary-color mt-0 mb-0">Show completed submissions</span>
-    <InputSwitch
+    <span class="app-primary-color">
+      {{ showCompleted ? 'Show active submissions' : 'Show completed submissions' }}
+    </span>
+    <ToggleSwitch
       v-model="showCompleted"
       class="ml-2 mt-0 mb-0 mr-2"
     />
   </div>
-  <TabView
+  <Tabs
     v-if="!loading"
-    v-model:activeIndex="activeTabIndex"
+    value="0"
```

Comment:
If `activeTabIndex` is no longer used, how are we defaulting to a tab based on the incoming query parameters?

Reply:
Fixed by setting top level Tabs component to watch for activeTabIndex again and setting the various tabs to numbers so it can respond to query parameters.